stripe/src/test/java/com/stripe/android/StripePaymentControllerTest.kt

```diff
@@ -589,25 +531,6 @@ internal class StripePaymentControllerTest {
         ).isTrue()
     }
 
-    @Test
-    fun handleSourceResult_withSuccessfulResult_shouldCallOnSuccess() {
```

Comment:
Why remove this test?

Reply:
oh per one of your previous comment I did some refactor to push the callbacks handling from `StripePaymentController` to `Stripe`, now the controller no longer has `#handlePaymentResult` `handleSetupResult` and `handleSourceResult` calls with callback, instead they are replaced with suspend funs to get the result directly. The corresponding tests are therefore removed.

The scenarios tested here are recreated in tests added in `StripePaymentAuthTest`